backend/lib/services/authorization.js

```diff
@@ -17,17 +17,13 @@
 'use strict'
 
 const _ = require('lodash')
-const kubernetes = require('../kubernetes')
-const Resources = kubernetes.Resources
-
-function Authorization ({ auth }) {
-  return kubernetes.authorization({ auth })
-}
+const { Resources } = require('../kubernetes-client')
 
 async function hasAuthorization (user, resourceAttributes) {
   if (!user) {
     return false
   }
+  const client = user.api
```

Comment:
rename api to client everywhere (and add comment that this is the client for the garden cluster. Same for the privilegedClient)

Reply:
✔️